src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperCloneCheck.java

```diff
@@ -37,7 +37,20 @@
  * <pre>
  * &lt;module name=&quot;SuperClone&quot;/&gt;
  * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Foo {
+ *
+ * public Object clone() {return null;} // violation, does not call super.clone()
```

Comment:
The xdoc has the methods indented but the javadoc does not. Please make them similar so it is also easier to read in the Javadoc. The methods should be similarly indented.

@strkkk Please keep an eye out for these type of things.